scripts/test/buildDeployTest.sh

```diff
@@ -70,15 +76,31 @@ cp -r "${PROJECT_ROOT}/java/system-test/target/system-test-${VERSION}-utility.ja
 echo "Starting Pre-deployment steps"
 ${PROJECT_ROOT}/scripts/deploy/pre-deployment.sh ${INSTANCE_ID} ${VPC} ${SUBNET} ${TABLE_NAME} ${TEMPLATE_DIR} ${GENERATED_DIR}
 
+END_CONFIGURE_DEPLOYMENT_TIME=$(record_time)
+echo "Configuring deployment finished at $(recorded_time_str "$END_CONFIGURE_DEPLOYMENT_TIME"), took $(elapsed_time_str "$END_BUILD_TIME" "$END_CONFIGURE_DEPLOYMENT_TIME")"
+
 echo "-------------------------------------------------------------------------------"
 echo "Deploying Stack"
 echo "-------------------------------------------------------------------------------"
 cdk -a "java -cp ${PROJECT_ROOT}/java/system-test/target/system-test-*-utility.jar sleeper.systemtest.cdk.SystemTestApp" deploy \
 --require-approval never -c testpropertiesfile=${GENERATED_DIR}/instance.properties -c validate=true "*"
 
+END_STACK_DEPLOYMENT_TIME=$(record_time)
+echo "Stack deployment finished at $(recorded_time_str "$END_STACK_DEPLOYMENT_TIME"), took $(elapsed_time_str "$END_CONFIGURE_DEPLOYMENT_TIME" "$END_STACK_DEPLOYMENT_TIME")"
+
 echo "-------------------------------------------------------------------------------"
 echo "Writing Random Data"
 echo "-------------------------------------------------------------------------------"
-CONFIG_BUCKET=$(cat ${GENERATED_DIR}/configBucket.txt)
 java -cp ${PROJECT_ROOT}/java/system-test/target/system-test-*-utility.jar \
 sleeper.systemtest.ingest.RunWriteRandomDataTaskOnECS ${INSTANCE_ID} system-test
+
+FINISH_TIME=$(record_time)
+echo "-------------------------------------------------------------------------------"
+echo "Finished"
+echo "-------------------------------------------------------------------------------"
+echo "Started at $(recorded_time_str "$START_TIME")"
+echo "Build finished at $(recorded_time_str "$END_BUILD_TIME"), took $(elapsed_time_str "$START_TIME" "$END_BUILD_TIME")"
+echo "Configuring deployment finished at $(recorded_time_str "$END_CONFIGURE_DEPLOYMENT_TIME"), took $(elapsed_time_str "$END_BUILD_TIME" "$END_CONFIGURE_DEPLOYMENT_TIME")"
+echo "Stack deployment finished at $(recorded_time_str "$END_STACK_DEPLOYMENT_TIME"), took $(elapsed_time_str "$END_CONFIGURE_DEPLOYMENT_TIME" "$END_STACK_DEPLOYMENT_TIME")"
+echo "Writing random data finished at $(recorded_time_str "$FINISH_TIME"), took $(elapsed_time_str "$END_STACK_DEPLOYMENT_TIME" "$FINISH_TIME")"
```

Comment:
I wonder if this would be better as "Starting of tasks to write random data finished at"? The call to RunWriteRandomDataTaskOnECS finishes when the tasks are deployed, rather than when the random data has actually been written.

Reply:
Done, thanks very much.